apps/modernization-ui/src/apps/page-builder/components/TargetQuestion/TargetQuestion.scss

```diff
@@ -0,0 +1,137 @@
+@use 'styles/borders' as borders;
+@use 'styles/colors';
+
+.target-question-tabs {
+    .tabs {
+        display: flex;
+        border-bottom: 2px solid var(--base-lighter, #dcdee0);
+        box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.09);
+        li {
+            display: inline-block;
+            cursor: pointer;
+            color: var(--base-black, #000);
+            font-family: Public Sans;
+            font-size: 16px;
```

Comment:
Suggestion: It is preferred to use rem over px for font sizing. We have also been using it for padding / margins.

Tip: In Figma you can change the size preferences by clicking the little settings icon:
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/be4ada6e-1cb1-459f-9dd3-b14d74ca7fd2)

Reply:
Nice i did not know figma showed rems!